public/locales/es/TexterTodo.json

```
@@ -0,0 +1,7 @@
+{
+  "all gone": "¡Todo se ha ido!",
+  "error releasing": "Error al publicar respuestas:",
```

Suggestion: I think `lieberar` (liberate/free) might be a better fit than `publicar` (post/publish)
```suggestion
  "error releasing": "Error al liberar respuestas:",
```